src/midje/parsing/2_to_lexical_maps/fakes.clj

```diff
@@ -13,18 +13,19 @@
 (defn- compiler-will-inline-fn? [var]
   (contains? (meta var) :inline))
 
+(defn- is-core?
+  "Is a variable from `clojure.core`, excluding `rand`.
+
+   `rand` is excluded because it is useful to fake and unused by Midje, so
+   there is no danger to faking it"
+  [var]
+  (let [var-meta (meta var)]
+    (and (not (= 'rand (:name var-meta)))
```

Comment:
Does making this exception feel right? It allows for flexibility but makes the rule "inconsistent"